src/bindings/c/CMakeLists.txt

```diff
@@ -4,6 +4,11 @@
 
 project(OpenVINO_C_API)
 
+if(CMAKE_COMPILER_IS_GNUCXX)
```

Comment:
I think you can just change the order of components in common CMake script.
As it was done in PR https://github.com/openvinotoolkit/openvino/pull/12666

Reply:
in this case they have to fix python as well. I suppose we can merge as is and remove these options later

Reply:
Yes, let's update it when we merge https://github.com/openvinotoolkit/openvino/pull/12666.

Reply:
@xipingyan I've merged my PR, please proceed with this one

Reply:
Done, please help review again.